artview/components/tools.py

```diff
@@ -58,6 +60,8 @@ def reset_tools(tooldict):
     -----
     Disconnects all tools.
     '''
+    import warnings
+    warnings.warn("this function is deprecated")
     for tool in tooldict:
         if tooldict[tool] is not None:
             tooldict[tool].disconnect()
```

Comment:
Fair enough. I have no problems with deprecating this. And you are right, this probably should be internal rather calling out to another location.